src/modules/notification/application/useEventNotifications.ts

```diff
@@ -2,19 +2,19 @@ import { ref } from 'vue';
 import type EventNotification from '../domain/entity/EventNotification';
 import EventNotificationService from '../domain/notification.service';
 import eventNotificationRepository from '../infrastructure/notification.repository';
-import eventNotificationStore from '../infrastructure/store/notification.store';
+import EventNotificationStore from '../infrastructure/store/notification.store';
```

Comment:
```suggestion
import eventNotificationStore from '../infrastructure/store/notification.store';
```

We're using an instance of the class here after all.